core/execution/market_activity_tracker.go

```diff
@@ -23,6 +25,7 @@ import (
 	proto "code.vegaprotocol.io/vega/protos/vega"
 )
 
+//go:generate go run github.com/golang/mock/mockgen -destination mocks/market_activity_tracker_mock.go -package mocks code.vegaprotocol.io/vega/core/execution EpochEngine
```

Comment:
isn't that one already in the engine.go file?

Reply:
indeed. removed from here